complete-application/src/main.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import App from "./App.tsx";
+import "./index.css";
+import { BrowserRouter } from "react-router-dom";
+import {
+  FusionAuthProvider,
+  FusionAuthProviderConfig,
+} from "@fusionauth/react-sdk";
+
+const config: FusionAuthProviderConfig = {
+  clientId: "e9fdb985-9173-4e01-9d73-ac2d60d1dc8e",
+  redirectUri: "http://localhost:3000",
+  serverUrl: "http://localhost:9011",
+  shouldAutoFetchUserInfo: true,
+  shouldAutoRefresh: true,
+  onRedirect: (state?: string) => {
+    console.log(`Redirect happened with state value: ${state}`);
```

Comment:
should we remove this?

Reply:
I added it as a way to demonstrate what that function does. Although it could be worth it to build out a more meaningful example. Let's not let that hold up this PR though, unless you think it's super important.

```
The startLogin and startRegister functions accept an optional string parameter: state. The value passed in will be passed to the onRedirect callback on your FusionAuthProviderConfig. Though you may pass any value you would like for the state parameter, it is often used to indicate which page the user was on before redirecting to login or registration, so that the user can be returned to that location after a successful authentication.
```

Reply:
Made this ticket https://github.com/FusionAuth/fusionauth-quickstart-javascript-react-web/issues/21